x-pack/plugin/otel-data/src/main/resources/component-templates/metrics-otel@mappings.yaml

```diff
@@ -3,6 +3,8 @@ _meta:
   description: Default mappings for the OpenTelemetry metrics index template installed by x-pack
   managed: true
 template:
+  settings:
+    index.mapping.ignore_malformed: true
```

Comment:
Late on this PR, but this is just a nit, so wouldn't change anything.

For traces and logs we set this implicitly via `index.mode: logsdb`, right? I wonder, maybe we should add this also to `logs-otel@mappings.yaml` and `traces-otel@mappings.yaml` explicitly. It would be better for readability - with OTel, we aim to never reject data, so I think it'd be worth the extra line, even if the current default is the same. Or maybe even move this to one of the central `yaml` files used by all signals.

Reply:
Right, for the other signals, it's implicit due to LogsDB. Good idea to move it to a central place used by all signals.

Reply:
Added to https://github.com/elastic/elasticsearch/issues/111877